packages/control-property-editor/src/panels/properties/HeaderField.tsx

```diff
@@ -81,8 +81,10 @@ export function HeaderField(headerFieldProps: Readonly<HeaderFieldProps>): React
             </UITooltip>
             <TextField
                 id={label}
+                data-testid={label}
                 value={value}
                 readOnly={true}
+                title={value}
```

Comment:
Is this information not required anymore

> 'The unique identifier within a page, either configured or automatically generated.'

Reply:
This information is related to the removed popup. We decided to remove the pop up, because it had hardcoded values and was not really helpful. Instead of the popup, we decided to add some tooltips for the values, because some of them are truncated.